ui/admin/app/components/form/worker/index.hbs

```diff
@@ -0,0 +1,87 @@
+<Rose::Form
+  @onSubmit={{@submit}}
+  @cancel={{@cancel}}
+  @disabled={{@model.isSaving}}
+  @error={{@model.errors.base}}
+  @showEditToggle={{if @model.isNew false true}}
+  as |form|
+>
+
+  {{#unless form.isEditable}}
+    <form.input
+      @name='type'
+      @value={{@model.type}}
+      @label={{t 'form.type.label'}}
+      @helperText={{t 'form.worker_type.help'}}
+      readonly={{true}}
+    />
+  {{/unless}}
+
+  <form.input
+    @name='name'
+    @type='name'
+    @value={{@model.name}}
+    @label={{t 'form.name.label'}}
+    @helperText={{t 'form.name.help'}}
+    @error={{@model.errors.name}}
+    readonly={{false}}
+    as |field|
+  >
+    {{#if @model.errors.name}}
+      <field.errors as |errors|>
+        {{#each @model.errors.name as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.input>
+
+  <form.textarea
+    @name='description'
+    @type='description'
+    @value={{@model.description}}
+    @label={{t 'form.description.label'}}
+    @helperText={{t 'form.description.help'}}
+    @error={{@model.errors.description}}
+    disabled={{@model.isSaving}}
+    readonly={{false}}
+    as |field|
+  >
+    {{#if @model.errors.description}}
+      <field.errors as |errors|>
+        {{#each @model.errors.description as |error|}}
+          <errors.message>{{error.message}}</errors.message>
+        {{/each}}
+      </field.errors>
+    {{/if}}
+  </form.textarea>
+
+  {{#unless form.isEditable}}
+    <form.input
+      @name='address'
+      @type='url'
+      @value={{@model.address}}
+      @label={{t 'form.address.label'}}
+      @helperText={{t 'form.worker_address.help'}}
+      readonly={{true}}
```

Comment:
Should we add error handling for `address` and `last seen` same as the other fields?